MobilityData/mobility-feed-api
# feat: Add GBFS feeds to the database

**Summary:**

This PR introduces the capability to add GBFS feeds to the database.

**Expected Behavior:**

The GBFS feeds are now integrated into the database as `Feed` and `GbfsFeed` entities. Each feed includes location information, external IDs referencing the `System ID` from `systems.csv`, and associated `GbfsVersion` entries. To avoid serving GBFS feeds temporarily, the `gbfs` data types have been filtered out of the `feeds` endpoint.

![image](https://github.com/user-attachments/assets/ecfe5518-38b2-410b-a3ff-c20dc0689e81)

**Out of Scope:** 

This PR does not include the addition of a trigger for updates from the `gbfs` repository. For more details, refer to https://github.com/MobilityData/mobility-feed-api/issues/675.

**Testing Tips:**
- Checkout the `feat/565-2` branch.
- Initialize the local database.
- Download `systems.csv` from the `gbfs` repository.
- Install dependencies and run: `python3 populate_db_gbfs.py --filepath "<path to systems.csv>"`.
- Validate the database content to ensure the feeds are correctly added.

Please make sure these boxes are checked before submitting your pull request - thanks!
